editor/block-toolbar/index.js

```diff
@@ -40,6 +76,54 @@ class BlockToolbar extends Component {
 		} ) );
 	}
 
+	onKeyDown( event ) {
+		if ( metaKeyPressed( event ) ) {
+			this.metaCount++;
+		}
+	}
+
+	stopPropagation( event ) {
+		// This is required to avoid messing up with the WritingFlow navigation
+		event.stopPropagation();
+	}
+
+	onKeyUp( event ) {
+		const isMeta = this.metaCount === 1;
+		this.metaCount = 0;
+
+		// Is there a better way to focus the selected block
+		const selectedBlock = document.querySelector( '.editor-visual-editor__block.is-selected' );
+		const tabbables = focus.tabbable.find( this.toolbar );
```

Comment:
How expensive is this? Shouldn't we try to run this and the above only when needed?